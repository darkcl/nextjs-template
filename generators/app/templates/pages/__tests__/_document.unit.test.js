import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';

import Document from '../_document';

describe('<Document />', () => {
  it('should render correctly', () => {
    const document = shallow(<Document />);
    expect(toJson(document)).toMatchSnapshot();
  });
});
