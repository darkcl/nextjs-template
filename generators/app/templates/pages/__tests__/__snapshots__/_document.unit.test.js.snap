// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Document /> should render correctly 1`] = `
<html>
  <Head>
    <meta
      content="initial-scale=1.0, width=device-width"
      key="viewport"
      name="viewport"
    />
    <link
      href="undefined/static/47cd76e43f74bbc2e1baaf194d07e1fa/images/favicon.png"
      rel="shortcut icon"
    />
    <link
      crossOrigin="anonymous"
      href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.3/css/bootstrap.min.css"
      integrity="sha384-Zug+QiDoJOrZ5t4lssLdxGhVrurbmBWopoEl+M6BdEfwnCJZtKxi1KgxUyJq13dy"
      rel="stylesheet"
    />
    <link
      href="undefined/static/47cd76e43f74bbc2e1baaf194d07e1fa/styles/bundle.css"
      rel="stylesheet"
    />
  </Head>
  <body>
    <Main />
    <NextScript />
  </body>
</html>
`;
